dss_metrics/metrics.py

```diff
@@ -51,6 +54,7 @@ def __init__(self, configs):
         self.filter = self.configs['filter']
         self.whitelist_patterns = utils.get_whitelist_keys()
         self.exit_flag = multiprocessing.Event()
+        self.logger = logging.getLogger('root')
```

Comment:
Use something like `logging.getLogger(__name__)` or define the application name in the top and use that app name